translate/test-ownership.js

```diff
@@ -7,20 +7,8 @@
  * This file is temporary and will be deleted after the
  * Vitest migration is complete.
  */
-export const ignoreFromJest = [
-  './src/modules/editor',
-  './src/hooks',
-  './src/modules/comments',
-  './src/modules/entities',
-  './src/modules/entitydetails',
-  './src/modules/history',
-  './src/modules/machinery',
-  './src/modules/otherlocales',
-  './src/modules/resourceprogress',
-  './src/modules/teamcomments',
-  './src/modules/unsavedchanges',
-  './src/modules/user',
+export const ingoreFromVitest = [
```

Comment:
Typo also needs fixing where this is referenced.
```suggestion
export const ignoreFromVitest = [
```

Reply:
fixed